code/game/gamemodes/events/biomass.dm

```diff
@@ -202,10 +202,11 @@
 		var/area/Hallway = locate(type)
 
 		for(var/turf/simulated/floor/Floor in Hallway.contents)
-			if(Floor.contents.len <= 0)
+			if(!is_blocked_turf(Floors))

```

Comment:
Surely you mean Floor here

Smooth variable names whoever made this originally